.github/workflows/main.yml

```diff
@@ -3,46 +3,46 @@ name: Main Build
 on:
   push:
     paths-ignore:
-      - "docs/**"
+    - docs/**
     branches:
-      - master
+    - master
   pull_request:
     types:
-      - opened
-      - synchronize
-      - reopened
+    - opened
+    - synchronize
+    - reopened
   workflow_dispatch:
 
 jobs:
   formatting:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: axel-op/googlejavaformat-action@v3
-        with:
+    - uses: actions/checkout@v3
+    - uses: axel-op/googlejavaformat-action@v3
+      with:
           # This version works will all IDEs including Eclipse.
-          version: 1.6
+        version: 1.6
           # args: "--dry-run --set-exit-if-changed"        
   build:
     runs-on: ubuntu-latest
     needs: [formatting]
     strategy:
       matrix:
-        java: [8, 11, 17]
+        java: [11, 17, 19]
```

Comment:
Can you please verify Payara 6 officially supports Java SE 19? As far as I am aware, it only supports up to Java SE 17?

Reply:
As I suspected, Payara 6 supports LTS releases only. Let's kindly stick to officially supported JDK versions: https://twitter.com/Payara_Fish/status/1610571608233500672?